packages/vuetify/src/components/VInfiniteScroll/VInfiniteScroll.tsx

```diff
@@ -292,6 +292,29 @@ export const VInfiniteScroll = genericComponent<VInfiniteScrollSlots>()({
         </Tag>
       )
     })
+
+    function reset () {
+      setStatus(props.side, 'ok')
```

Comment:
It won't work for `both`. I guess `setStatus` should be updated to accept 3rd option to set both `start` and `end`.

Reply:
Oh alright I see, I completely missed that. I will be revisiting this shortly and updating everything that requires it